scripts/transform_zipcodes.rb

```diff
@@ -0,0 +1,16 @@
+require 'csv'
```

Comment:
Note: this PR adds two files in `data/`. The first is the CSV we were using previously for zipcode data, and the second is the transformed version which is smaller and easier to work with in the browser, cause it's JSON. This script does the transformation; including it in case we need to run it again.